tests/test_dependency_rendering_e2e.py

```diff
@@ -106,17 +111,34 @@ async def test_multiple_component_dependencies(self):
         data = await page.evaluate(test_js)
 
         # Check that the actual HTML content was loaded
-        self.assertInHTML(
-            """
-            <div class="outer">
-                Variable: <strong class="inner">variable</strong>
-                XYZ: <strong class="other">variable_inner</strong>
-            </div>
-            <div class="my-style">123</div>
-            <div class="my-style2">xyz</div>
-            """,
+        self.assertRegex(
             data["bodyHTML"],
-            count=1,
+            # <div class="outer" data-djc-id-10uLMD>
+            #     Variable:
+            #     <strong class="inner" data-djc-id-DZEnUC>
+            #         variable
+            #     </strong>
+            #     XYZ:
+            #     <strong class="other" data-djc-id-IYirHK>
+            #         variable_inner
+            #     </strong>
+            # </div>
+            # <div class="my-style">123</div>
+            # <div class="my-style2">xyz</div>
+            re.compile(
```

Comment:
Had to update some tests to check via regex because in the E2E tests the component IDs are random.